src/Campaigns/Actions/AssociateCampaignPageWithCampaign.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Give\Campaigns\Actions;
+
+use Exception;
+use Give\Campaigns\Models\Campaign;
+use Give\Campaigns\Models\CampaignPage;
+
+class AssociateCampaignPageWithCampaign
+{
+    /**
+     * @throws Exception
+     */
+    public function __invoke(CampaignPage $campaignPage){
+        $campaign = Campaign::find($campaignPage->campaignId);
```

Comment:
Friendly reminder that our models are nullable and will cause a fatal error when accessing properties on null.  This one looks safe but might as well ask if `campaignId` from the campaignPage could potentially be null or campaign not found. If so, we should make sure `$campaign` exists before accessing properties.

Reply:
Resolved https://github.com/impress-org/givewp/pull/7721/commits/1466a32c7a87efe2edd8911cf3d38cf7f7771e21